src/components/views/messages/TextualBody.js

```diff
@@ -59,10 +64,23 @@ module.exports = React.createClass({
 
             // track whether the preview widget is hidden
             widgetHidden: false,
+
+            editMode: false,
         };
     },
 
+    onAction(payload) {
+        // This isn't very nice -- but make sure it is the textualBody for the same
+        // mxEvent as the MessageContextMenu was generated for
+        // localize subscribe/dispatch?
+        if (payload.action === 'edit' && payload.mxEvent === this.props.mxEvent) {
+            this.state.editMode = !this.state.editMode
+            this.forceUpdate()
```

Comment:
First time using React sorry about this =( 

I assume there is somewhere I should register `editMode` so that changes to it trigger re-render.. (will-fix)